### DISCUSSION TITLE
Google Coolab not works for me

### DISCUSSION THREAD

**Discussion by Ruskoff9966** [lllyasviel/Fooocus#2584] [General]
Hello when i start the google coolab program It doesn't launch and leaves me waiting for the python launch executive any one can help please. thanks you

**Comment by mashb1t** [comment]
you need to press the play button in Colab to start the execution, then wait for the startip to finish and click the gradio.live link shown in the output.

**Comment by Ruskoff9966** [comment]
thanks a lot i just waiting many times and it works thanks
